### PR TITLE
[Fix] Making installs on examples "shell safe"

### PR DESCRIPTION
**Pull request by murilommen**
## Description

Adding quotes for installations to run on different shell environments

### General Checklist

* [ ] Tests added for this feature/bug
      if it was a bug, test must cover it.
* [ ] Conform by the style guides, by using formatter
* [x] Documentation updated
* [ ] (optional) Please add a label to your PR

### CODE REVIEW

File: python/examples/basic/Getting_Started.ipynb [hunk 1]
```diff
@@ -65,7 +65,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install -q whylogs --pre"
+    "! pip install -q whylogs --pre"
```
Comment (andyndang) [inline python/examples/basic/Getting_Started.ipynb]: It's a bit strange to have extra spaces here?

Comment (murilommen) [reply python/examples/basic/Getting_Started.ipynb]: my jupyter notebook complains about not having the extra space

Comment (murilommen) [reply python/examples/basic/Getting_Started.ipynb]: but i can remove it

**Comment by jamie256** [comment]
Can we update to resolve the conflicts? Otherwise looks fine.

**Review by TheMellyBee — APPROVED** [review]
LGTM

**Pull request merged**